docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -1,261 +1,448 @@
 ---
-title: Cert Authority Rotation
-description: How to rotate Teleport's certificate authority
+title: Certificate Authority Rotation
+description: Describes how to rotate Teleport's certificate authorities.
+tocDepth: 3
 ---
 
+Various components of a Teleport cluster authenticate to one another using
+either X.509 or SSH certificates. To verify certificates, Teleport maintains
+several certificate authorities. You can rotate Teleport CAs to prevent
```

Comment:
I would say that Teleport uses the certificate authorities to _issue certificates_. Verification against the CA is done by the target resources (Windows desktop, database, SSH node, etc)